midend/coverage.h

```diff
@@ -8,7 +8,12 @@
 #include "ir/visitor.h"
 #include "lib/source_file.h"
 
-namespace P4Tools {
+/// This file is a collection of utilities for coverage tracking in P4 programs. Currently, this
```

Comment:
The question is whether this file should stile in tools or move to the mid end. If there is no use for it, it can stay in tools.